### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by rootvar**
Added following icons:
Screen Shot 2019-04-16 at 4 20 07 PM
Screen Shot 2019-04-16 at 4 20 11 PM
Screen Shot 2019-04-16 at 4 19 29 PM
Screen Shot 2019-04-16 at 4 19 32 PM
Screen Shot 2019-04-16 at 4 19 38 PM
Screen Shot 2019-04-16 at 4 19 40 PM
Screen Shot 2019-04-16 at 4 19 53 PM
Screen Shot 2019-04-16 at 4 19 59 PM
Screen Shot 2019-04-16 at 4 56 36 PM

All optimized with svgo.
Also ordered all imports alphabetically.

### CODE REVIEW

File: packages/gestalt/src/icons/duplicate.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...727273,10.2985701 9.33779352,10.9090909 10.0909091,10.9090909 L13.0909091,10.9090909 L13.0909091,13.9090909 C13.0909091,14.6622065 13.7014299,15.2727273 14.4545455,15.2727273 C15.207661,15.2727273 15.8181818,14.6622065 15.8181818,13.9090909 L15.8181818,10.9090909 L18.8181818,10.9090909 C19.5709091,10.9090909 20.1818182,10.2981818 20.1818182,9.54545455 M24,2.45454545 L24,16.6363636 C24,17.9919717 22.9010626,19.0909091 21.5454545,19.0909091 L7.36363636,19.0909091 C6.00802834,19.0909091 4.90909091,17.9919717 4.90909091,16.6363636 L4.90909091,2.45454545 C4.90909091,1.09893743 6.00802834,2.42230478e-16 7.36363636,0 L21.5454545,0 C22.9010626,0 24,1.09893743 24,2.45454545 M17.4545455,22.6363636 C17.4545455,23.3890909 16.8436364,24 16.0909091,24 L6.81818182,24 C3.05434854,23.9957916 0.00420842706,20.9456515 0,17.1818182 L0,7.90909091 C2.42230478e-16,7.15597534 0.610520796,6.54545455 1.36363636,6.54545455 C2.11675193,6.54545455 2.72727273,7.15597534 2.72727273,7.90909091 L2.72727273,17.1818182 C2.72967779,19.4401679 4.55983207,21.2703222 6.81818182,21.2727273 L16.0909091,21.2727273 C16.8436364,21.2727273 17.4545455,21.8836364 17.4545455,22.6363636" fill="#8E8E8E" fill-rule="nonzero"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/duplicate.svg]: @rootvar are these from production design team? They look like they were not optimized
image

Comment (rootvar) [reply packages/gestalt/src/icons/duplicate.svg]: They are from our designer (exported from Sketch). Do we use svgo to optimize svgs?

Comment (pfarejowicz) [reply packages/gestalt/src/icons/duplicate.svg]: Yup, but you shouldn’t need to do anything. Can you or your designer please ping @jaymarsh or @scottandreae with the icon requests? They will create the optimized pds approved asset and those are the ones that go into gestalt (we don’t use ones from sketch). There’s a specific form for your designer to fill out under the PD project in Jira

File: packages/gestalt/src/icons/duplicate.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M20.182 9.545c0-.752-.611-1.363-1.364-1.363h-3v-3a1.364 1.364 0 1 0-2.727 0v3h-3a1.364 1.364 0 1 0 0 2.727h3v3a1.364 1.364 0 1 0 2.727 0v-3h3c.753 0 1.364-.61 1.364-1.364M24 2.455v14.181a2.455 2.455 0 0 1-2.455 2.455H7.364a2.455 2.455 0 0 1-2.455-2.455V2.455A2.455 2.455 0 0 1 7.364 0h14.181A2.455 2.455 0 0 1 24 2.455m-6.545 20.181c0 .753-.611 1.364-1.364 1.364H6.818A6.826 6.826 0 0 1 0 17.182V7.909a1.364 1.364 0 1 1 2.727 0v9.273a4.095 4.095 0 0 0 4.091 4.09h9.273c.753 0 1.364.612 1.364 1.364" fill="#8E8E8E"/></svg>
```
Comment (rootvar) [inline packages/gestalt/src/icons/duplicate.svg]: @pfarejowicz optimized all icons with svgo. Thanks for the hint!

Comment (pfarejowicz) [reply packages/gestalt/src/icons/duplicate.svg]: Can we still check with the PDS folks? Icons here shouldn’t come from sketch + svgo manually but instead through production requests

Comment (rootvar) [reply packages/gestalt/src/icons/duplicate.svg]: yes, I pinged PDS

File: packages/gestalt/src/icons/ad-group.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M9 11a1 1 0 0 1-1-1V1a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1v9a1 1 0 0 1-1 1H9zm1-2h4V7h-4v2zm10 4a1 1 0 0 1 1 1v9a1 1 0 0 1-1 1h-6a1 1 0 0 1-1-1v-9a1 1 0 0 1 1-1h6zm-5 9h4v-2h-4v2zm-5-9a1 1 0 0 1 1 1v9a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-9a1 1 0 0 1 1-1h6zm-5 9h4v-2H5v2z" fill="#8E8E8E" fill-rule="evenodd"/></svg>
```
Comment (rootvar) [inline packages/gestalt/src/icons/ad-group.svg]: @pfarejowicz I have updated all icons with PDS provided versions. Thanks!

Comment (rootvar) [reply packages/gestalt/src/icons/ad-group.svg]: @user-wkang thanks for your help!

**Pull request merged**